### PR TITLE
Add developerConsolePublicURL to config

### PR DESCRIPTION
**Pull request by spadgett**
@deads2k PTAL
cc @jwforres

### CODE REVIEW

File: cmd/bridge/config.go [hunk 1]
```diff
@@ -41,6 +41,9 @@ type ClusterInfo struct {
 	ConsoleBaseAddress string `yaml:"consoleBaseAddress"`
 	ConsoleBasePath    string `yaml:"consoleBasePath"`
 	MasterPublicURL    string `yaml:"masterPublicURL"`
+	// DeveloperPublicConsoleURL is an optional URL of the developer console. If specified, a context switcher is
+	// added to the masthead with a link to the developer console. This option will be removed in a future release.
+	DeveloperPublicConsoleURL string `yaml:"developerConsolePublicURL"`
```
Comment (deads2k) [inline cmd/bridge/config.go]: new field lgtm

Comment (deads2k) [reply cmd/bridge/config.go]: > new field lgtm

remember that you'll need to leave it inert for one release to allow a clean update and rollback.

**Comment by spadgett** [comment]
/assign @benjaminapetersen 

@benjaminapetersen Mainly just renaming the existing vars.

**Comment by spadgett** [comment]
jenkins rebuild

**Comment by spadgett** [comment]
@deads2k When you have time, can you look at the corresponding change here as well?

https://github.com/openshift/api/pull/76

Each console links to the other, so each has a new property in its respective config.

**Pull request merged**